en/intro/conventions.rst

```diff
@@ -151,6 +167,22 @@ See `awesome list recommendations
 <https://github.com/FriendsOfCake/awesome-cakephp/blob/master/CONTRIBUTING.md#tips-for-creating-cakephp-plugins>`__
 for details.
 
+Constants
+=========
+
+Always keep the same pattern, as a suggestion, in capital letters and underlines:
+"MIN_USER_AGE", "MY_CONSTANT_PERSONAL".
+
+Variables
+=========
+
+Always use the same pattern throughout the camelCase application as in $userComment, $total, $countPerson.
+
+The first letter is uppercase if the variable represents an object:
```

Comment:
We no longer follow this convention in 3.x+.

Reply:
To clarify: This is only used still where the object is a special locator based class attribute

$this->loadModel('Foo') and $this->Foo

but

$foo = new Foo();

etc.